### PR TITLE
Make the fallback path exception free and support IPv6 only setup

### PR DESCRIPTION
**Pull request by rjschwei**
- Handle IPv4 setup as optional This was previously required. All IPv4 configuration settings are treated as optional. Missing configurations no longer trigger exceptions.
- Make the fallback path exception free Previously not configuring client IP ranges that serve as fallback path for region lookup if no region hint was provided or the region was not found would trigger and exception. The fallback path is now exception free supporting a setup where the regionHint is mandatory.
- Add test config files to test the server locally
- Add test instructions for local testing

### CODE REVIEW

File: srv/www/regionService/regionInfo.py [hunk 1]
```diff
@@ -84,32 +84,39 @@ def create_smt_region_map(conf):
...
+            info_msg += '%s No IPv6 address based fallback possible' % section
+            logging.info(info_msg)
         try:
+            region_smt_ips = None
             region_smt_ips = region_data_cfg.get(section, 'smt-server-ip')
         except Exception:
-            logging.error('Missing smt-server-ip data in section %s' % section)
-            sys.exit(1)
+            info_msg = 'smt-server-ip data in section %s not ' % section
+            info_msg += 'configured Update servers cannot be reached over IPv4'
```
Comment (jesusbv) [inline srv/www/regionService/regionInfo.py]: s/configured/configured.
or 
s/configured/configured\n

File: srv/www/regionService/regionInfo.py [hunk 2]
```diff
@@ -84,32 +84,39 @@ def create_smt_region_map(conf):
...
+            logging.info(info_msg)
         try:
             region_smt_ipsv6 = None
             region_smt_ipsv6 = region_data_cfg.get(section, 'smt-server-ipv6')
         except Exception:
-            # IPv6 addresses are optional not all cloud frameworks support IPv6
-            pass
+            info_msg = 'smt-server-ipv6 data in section %s not ' % section
+            info_msg += 'configured Update servers cannot be reached over IPv6'
```
Comment (jesusbv) [inline srv/www/regionService/regionInfo.py]: ditto

File: srv/www/regionService/regionInfo.py [hunk 3]
```diff
@@ -84,32 +84,39 @@ def create_smt_region_map(conf):
...
         try:
+            region_public_ip_ranges = ''
             region_public_ip_ranges = region_data_cfg.get(
                 section,
                 'public-ips'
             )
         except Exception:
-            logging.error('Missing public-ips data in section %s' % section)
-            sys.exit(1)
+            info_msg = 'public-ips data not configured in section %s' % section
```
Comment (jesusbv) [inline srv/www/regionService/regionInfo.py]: s/%s/%s.
or s/%s/%s\n

**Review by jesusbv — APPROVED** [review]
other than minor changes on the log message, LGTM

**Review by bear454 — APPROVED** [review]
Thank you for adding a test process :tada:

**Pull request merged**